bin/tithe

```diff
@@ -66,92 +43,97 @@ Table.defaults.marks = {
   , mb: "┴"
   , mm: "┼"
 };
+// Logger configs
+Logger.config.displayDate = false;
+Logger.config.logLevel = 4;
+let tithe = null;
 
+let initTithe = () => tithe = new Tithe(parser.options.P.value);
 
-/*!
- * log
- *
- * @name log
- * @function
- * @param {Error} err The error.
- * @param {String} data The info message.
- * @return {undefined}
- */
-function log(err, data) {
-    if (err) { return Logger.log(err.stack || err, "error"); }
-    Logger.log(data, "info");
-}
-
-// Handle the custom path
-if (tFileOpt.is_provided) {
-    tFileOpt.value = Fs.absolute(tFileOpt.value);
-    Tithe.path = tFileOpt.value;
-}
-
-// Insert
-if (insertOpt.is_provided) {
-
-    // Description
-    if (!descriptionOpt.value) {
-        return log("Description is required.");
+let parser = new Tilda(`${__dirname}/../package.json`, {
```

Comment:
@lcristianiim Here you can see how migration from `clp` to `tilda` works. :grin: